packages/model-viewer/src/features/controls.ts

```diff
@@ -34,22 +34,28 @@ import {timeline} from '../utilities/animation.js';
 const PROMPT_ANIMATION_TIME = 5000;
 
 // For timing purposes, a "frame" is a timing agnostic relative unit of time
-// and a "value" is a target value for the keyframe.
-const wiggle = timeline(0, [
-  {frames: 5, value: -1},
-  {frames: 1, value: -1},
-  {frames: 8, value: 1},
-  {frames: 1, value: 1},
-  {frames: 5, value: 0},
-  {frames: 18, value: 0}
-]);
-
-const fade = timeline(0, [
-  {frames: 1, value: 1},
-  {frames: 5, value: 1},
-  {frames: 1, value: 0},
-  {frames: 6, value: 0}
-]);
+// and a "value" is a target value for the Frame.
+const wiggle = timeline({
```

Comment:
Out of curiosity, based on what we choose these numbers and number of frames?

Reply:
These didn't change; the default prompt was designed with input from the Shopify UX folks a few years ago.